deploy/scripts/src/mobile/utils/backend.ts

```diff
@@ -0,0 +1,106 @@
+import http from 'k6/http'
+import { group } from 'k6'
+import { uuidv4 } from './k6/k6-utils'
+import { getBackendUrl } from './url'
+import { parseTestClientResponse, postTestClientStart } from './test-client'
+import { isStatusCode200, isStatusCode201 } from './assertions'
+
+export function postVerifyAuthorizeRequest (): string {
+  let verifyUrl: string
+  group('POST test client /start', () => {
+    const testClientRes = postTestClientStart()
+    isStatusCode201(testClientRes)
+    verifyUrl = parseTestClientResponse(testClientRes, 'ApiLocation')
+  })
+
+  return group('POST /verifyAuthorizeRequest', () => {
+    const verifyRes = http.post(verifyUrl)
+    isStatusCode200(verifyRes)
+    return verifyRes.json('sessionId') as string
+  })
+}
+
+export function postResourceOwnerDocumentGroups (sessionId: string): void {
+  group('POST /resourceOwner/documentGroups', () => {
+    const documentGroupsData = {
+      resourceOwner: {
+        documentGroups: [
+          {
+            groupName: 'Photo Identity Document',
+            allowableDocuments: ['NFC_PASSPORT']
+          }
+        ]
+      }
+    }
+    const documentGroupsUrl = getBackendUrl(`/resourceOwner/documentGroups/${sessionId}`)
+    const res = http.post(documentGroupsUrl, JSON.stringify(documentGroupsData))
+    isStatusCode200(res)
+  })
+}
+
+export function getBiometricTokenV2 (sessionId: string): void {
+  group('GET /biometricToken/v2', () => {
+    const biometricTokenUrl = getBackendUrl('/biometricToken/v2', { authSessionId: sessionId })
+    const res = http.get(biometricTokenUrl, {
+      tags: { name: 'Get Biometric Token' }
```

Comment:
Are we expecting a tag here? We don't seem to be using them in other requests

Reply:
I think this was left over from us copying across some FE stuff. Since we've discussed already not adding things where we don't know what it does and without knowing the value of it, I've removed it